android/sdk/src/main/java/org/jitsi/meet/sdk/JitsiMeetActivity.java

```diff
@@ -227,4 +227,9 @@ public void onConferenceTerminated(Map<String, Object> data) {
     public void onConferenceWillJoin(Map<String, Object> data) {
         JitsiMeetLogger.i("Conference will join: " + data);
     }
+
```

Comment:
This change was intentional? We don't use this JitsiMeetActivity, do we?

Reply:
We don't, but it still must implement the JitsiMeetViewListener interface. If we don't follow the changes there the build would fail.